src/net.h

```diff
@@ -41,6 +41,8 @@ namespace boost {
 static const int PING_INTERVAL = 2 * 60;
 /** Time after which to disconnect, after waiting for a ping response (or inactivity). */
 static const int TIMEOUT_INTERVAL = 20 * 60;
+/** Run the feeler connection loop once every 2 minutes or 120 seconds. **/
+static const int FEELER_INTERVAL = 120;
```

Comment:
@sipa @laanwj Given this current apparent convention causes a waste of CPU compiling code that doesn't need compiling, can we move some of the net.h code to net.cpp (or other applicable) places in a future pull request?